packages/wonder-stuff-server/tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-  "exclude": ["dist"],
+  "exclude": ["dist", "**/*.flowtest.ts"],
```

Comment:
I'm fairly certain that typescript flowtest files make no sense.

Reply:
I want to convert these to `.typetest.ts` files instead of deleting them. I'll do that in a separate PR.